deck.md

```diff
@@ -406,19 +459,43 @@ test('GET /users', async t => {
 
 ---
 
+# Step 5: Trying it out
+
+#### Run the tests
+
+```a
+yarn test
+
+$ tap
+ PASS  test/index.test.js 2 OK 88.011ms
+ PASS  test/login.test.js 5 OK 121.376ms
+
+🌈 SUMMARY RESULTS 🌈
+
+Suites:   2 passed, 2 of 2 completed
+Asserts:  7 passed, of 7
+Time:     721.846ms
+----------|----------|----------|----------|----------|-------------------|
+File      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
+----------|----------|----------|----------|----------|-------------------|
+All files |        0 |        0 |        0 |        0 |                   |
+----------|----------|----------|----------|----------|-------------------|
+✨  Done in 2.85s.
+```
+
+---
+
 class: branded
 
-# Step 3: Validation and Authentication
+# Step 6: Validation
 
 - Route validation internally relies upon [Ajv](https://www.npmjs.com/package/ajv), which is a high-performance JSON Schema validator
```

Comment:
not sure we should add `JWT_SECRET` in the `Validation` step 🤔

Reply:
![Screenshot 2021-02-02 at 16 27 31](https://user-images.githubusercontent.com/272483/106622129-91ff3c80-6573-11eb-8ba4-6bf20801379c.png)

Reply:
This step seems to be more "configuration" then "validation" or routes 🤔

Reply:
Latest updates have resolved this